adapters/Facebook/FacebookAdapter/OpenBidding/GADFBInterstitialRenderer.m

```diff
@@ -96,6 +99,11 @@ - (void)interstitialAdDidLoad:(FBInterstitialAd *)interstitialAd {
 }
 
 - (void)interstitialAd:(FBInterstitialAd *)interstitialAd didFailWithError:(NSError *)error {
+  if (_presentCalled) {
+    NSLog(@"Received a Facebook SDK error during presentation: $@", error.localizedDescription);
+    _adLoadCompletionHandler(nil, error);
```

Comment:
Can we drop rows 104-105 as we still send the error on 107?

Reply:
It was a code error, and it was corrected. Line 104 should be [_adEventDelegate didFailToPresentWithError:error];